### PR TITLE
Update COMPILE.md to disable shared lib in compiling protobuf.

### PR DESCRIPTION
**Pull request by voidzcy**

### CODE REVIEW

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* User @voidzcy isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf

_Regards,
CLA GitHub bot_

**Comment by voidzcy** [comment]
Ping for CLA

**Comment by voidzcy** [comment]
Ping for CLA

**Comment by voidzcy** [comment]
Ping for CLA

**Comment by voidzcy** [comment]
Ping for CLA

File: COMPILING.md [hunk 1]
```diff
@@ -46,15 +46,15 @@ $ git clone https://github.com/google/protobuf.git
...
 $ make
 $ make check
 $ sudo make install
 ```
 
 If you are comfortable with C++ compilation and autotools, you can specify a
 ``--prefix`` for Protobuf and use ``-I`` in ``CXXFLAGS``, ``-L`` in
-``LDFLAGS``, ``LD_LIBRARY_PATH``, and ``PATH`` to reference it. The
+``LDFLAGS`` to reference it. The
 environment variables will be used when building grpc-java.
```
Comment (ejona86) [inline COMPILING.md]: Ideally this line would have been reflowed and the comma would have been replaced with "and". But whatever.

**Pull request merged**